src/components/AddStack.js

```diff
@@ -24,4 +25,8 @@ class AddNewStack extends Component {
   }
 }
 
+AddNewStack.PropTypes = {
```

Comment:
`AddNewStack.PropTypes` should be `AddNewStack.propTypes`. In addition, this component doesn't really have props -- it only has state.